openslides/core/config.py

```diff
@@ -122,6 +138,7 @@ def __setitem__(self, key, value):
         try:
             db_value = ConfigStore.objects.get(key=key)
         except ConfigStore.DoesNotExist:
+            # This should never happen. All DB entries are created at startup.
```

Comment:
I removed the try-except block. It is really impossible to get this error message, because of the try-except block at the beginning of the function.